ibm/service/satellite/data_source_ibm_satellite_cluster.go

```diff
@@ -241,7 +246,7 @@ func dataSourceIBMSatelliteClusterRead(d *schema.ResourceData, meta interface{})
 	d.Set("state", *clusterFields.State)
 	d.Set("status", *clusterFields.Status)
 	d.Set("workers", workers)
-
```

Comment:
Can we add condition check and then set the value 

```
if clusterFields.InfrastructureTopology != nil {
d.Set("infrastructure_topology", *clusterFields.InfrastructureTopology)
}
```
or The API always gives back some value to the InfrastructureTopology?